src/button/styles.scss

```diff
@@ -164,7 +164,3 @@
     inset-inline: 0;
   }
 }
-
-.link.disabled {
-  pointer-events: none;
```

Comment:
This style does not seem to do anything useful, since preventing a user action for disabled link button is implemented in the click handler. And this code prevents triggering the disabled reason tooltip